docs/aws/README.md

```diff
@@ -81,15 +89,14 @@ helm upgrade --install karpenter karpenter/karpenter \
   --namespace karpenter --set serviceAccount.create=false
 ```
 
-### (Optional) Enable Verbose Logging
-```bash
-kubectl patch deployment karpenter-controller \
-    -n karpenter --type='json' \
-    -p='[{"op": "replace", "path": "/spec/template/spec/containers/0/args", "value": ["--verbose"]}]'
-```
+## Provisioner
+
+A single karpenter provisioner is capable of handling many different pod shapes. In other words, karpenter eliminates the need to manage many different node groups. Karpenter makes scheduling and provisioning decisions based on pod attributes such as labels and affinity. 
```

Comment:
If clarify this to "node scheduling properties, such as instance type and labels" rather than pod shapes.

Reply:
Also -- karpenter doesn't currently support affinity. This is coming in v0.4